payloads/v9/stageInstance.ts

```diff
@@ -0,0 +1,58 @@
+import type { Snowflake } from '../../globals';
+import type { APIGuildMember } from './guild';
+
+/**
+ * https://discord.com/developers/docs/resources/stage-instance#stage-instance-object
+ */
+export interface APIStageInstance {
+	/**
+	 * The id of the stage instance
+	 */
+	id: Snowflake;
+	/**
+	 * The guild id of the associated stage channel
+	 */
+	guild_id: Snowflake;
+	/**
+	 * The id of the associated stage channel
+	 */
+	channel_id: Snowflake;
+	/**
+	 * The topic of the stage instance (1-120 characters)
+	 */
+	topic: string;
+	/**
+	 * The privacy level of the stage instance
+	 *
+	 * See https://discord.com/developers/docs/resources/stage-instance#stage-instance-object-privacy-level
+	 */
+	privacy_level: StageInstancePrivacyLevel;
+	/**
+	 * Whether or not stage discovery is disabled
+	 */
+	discoverable_disabled: boolean;
+}
+
+/**
+ * https://discord.com/developers/docs/resources/stage-instance#stage-instance-object-privacy-level
+ */
+export const enum StageInstancePrivacyLevel {
+	/**
+	 * The stage instance is visible publicly, such as on stage discovery
+	 */
+	PUBLIC = 1,
+	/**
+	 * The stage instance is visible to only guild members
+	 */
+	GUILD_ONLY,
+}
+
+export interface APIInviteStageInstance {
+	topic: string;
```

Comment:
Are there no docs for this?

Reply:
I must have forgot, I'll add them. Also, am I suppose to write the same stuff for `v8` too? I was unsure so I just added the changes only in `v9`.